test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt

```diff
@@ -225,7 +225,7 @@ class AndroidArgsTest {
           test: $testApk
           device:
           - model: walleye
-            version: 18
+            version: 28
```

Comment:
There's no walleye/28
Did you mean 27?

Reply:
This test is about androidArgs incompatibleModel. 28 is also invalid. I will revert to 18 nevertheless.